src/Host.cpp

```diff
@@ -1378,19 +1378,33 @@ void Host::raiseEvent(const TEvent& pE)
         return;
     }
 
+    static QString star = QString("*");
```

Comment:
```suggestion
    static QString star = QStringLiteral("*");
```

This makes it free (QString is created at compile time) :)

Reply:
Thanks. Updated accordingly.